### DISCUSSION TITLE
Custom xem url

### DISCUSSION THREAD

**Discussion by Guslletas** [pymedusa/Medusa#10540] [Ideas & Feature requests]
I've set up my own xem server and I'd love if there was an option(even if it was just in some config file and not in the gui) to set up a custom xem url so I don't have to change it manually every update

**Comment by p0psicles** [comment]
I'll add an option to the config.ini

**Reply by p0psicles** [reply]
So I guess it's only for the base url?

**Reply by Guslletas** [reply]
yeah, that would be enough

**Reply by p0psicles** [reply]
Added to the develop branch.
You'll have to save `a` config once. Then it'll add the key `xem_url` to the config.ini. Stop medusa, overwrite the value and start.

**Reply by Guslletas** [reply]
Thanks!!
